src/qt/guiutil.cpp

```diff
@@ -407,10 +407,6 @@ void bringToFront(QWidget* w)
 {
     if (w) {
         if (QGuiApplication::platformName() == "wayland") {
-            auto flags = w->windowFlags();
-            w->setWindowFlags(flags|Qt::WindowStaysOnTopHint);
-            w->show();
-            w->setWindowFlags(flags);
```

Comment:
This code was introduced in https://github.com/bitcoin-core/gui/pull/831.

cc @pablomartin4btc